packages/client/src/comp/states/BaseRegistration.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react'
+import { RegistrationStatus } from '../../common'
+import styled from 'styled-components'
+
+const I = styled.i`
+  color: var(--primary);
+  &:hover { cursor: help; }
+`
+
+const MarginSpan = styled.span`
+  display: block;
+  margin-bottom: 20px;
+`
+
+const Red = styled.span`
+  color: var(--danger);
+`
+
+export type BaseRegistrationStatus = RegistrationStatus | 'Error' | 'Loading'
```

Comment:
I don't think you have to extend this type: you can just pass in `RegistrationStatus`

Reply:
OK -- I think you're right, we should keep the extended type.